packages/mc-scripts/src/webpack-loaders/message-loader.spec.ts

```diff
@@ -0,0 +1,77 @@
+import path from 'path';
+import { createFsFromVolume, Volume } from 'memfs';
+import webpack, { type Stats } from 'webpack';
+
+const compiler = (fixture: string): Promise<Stats> => {
+  const compiler = webpack({
+    context: __dirname,
+    entry: `./${fixture}`,
```

Comment:
Nit: can we move the fixtures within a `fixtures` folder or something?

```suggestion
    entry: `./fixtures/${fixture}`,
```